internal/impl/confluent/processor_schema_registry_decode.go

```diff
@@ -65,7 +65,11 @@ This processor decodes protobuf messages to JSON documents, you can read more ab
 		Field(service.NewBoolField("avro_raw_json").
 			Description("Whether Avro messages should be decoded into normal JSON (\"json that meets the expectations of regular internet json\") rather than https://avro.apache.org/docs/current/specification/_print/#json-encoding[Avro JSON^]. If `true` the schema returned from the subject should be decoded as https://pkg.go.dev/github.com/linkedin/goavro/v2#NewCodecForStandardJSONFull[standard json^] instead of as https://pkg.go.dev/github.com/linkedin/goavro/v2#NewCodec[avro json^]. There is a https://github.com/linkedin/goavro/blob/5ec5a5ee7ec82e16e6e2b438d610e1cab2588393/union.go#L224-L249[comment in goavro^], the https://github.com/linkedin/goavro[underlining library used for avro serialization^], that explains in more detail the difference between the standard json and avro json.").
 			Advanced().Default(false)).
-		Field(service.NewURLField("url").Description("The base URL of the schema registry service."))
+		Field(service.NewURLField("url").
+			Description("The base URL of the schema registry service.")).
+		Field(service.NewBoolField("emit_default_values").
```

Comment:
Can we instead create a nested config object and add a boolean field for `emit_defaults`? I'd like to namespace these as over time I expect to have more protobuf knobs available.

That will look something like this:

```
service.NewObjectField("protobuf", service.NewBoolField("emit_defaults"))
```